target/linux/ramips/files/drivers/mmc/host/mtk-mmc/sd.c

```diff
@@ -1966,14 +1966,16 @@ static irqreturn_t msdc_irq(int irq, void *dev_id)
 		} else if ((intsts & MSDC_INT_RSPCRCERR) || (intsts & MSDC_INT_ACMDCRCERR)) {
 			if (intsts & MSDC_INT_ACMDCRCERR)
 				IRQ_MSG("XXX CMD<%d> MSDC_INT_ACMDCRCERR", cmd->opcode);
-			else
+			else {
 				IRQ_MSG("XXX CMD<%d> MSDC_INT_RSPCRCERR", cmd->opcode);
+			}
```

Comment:
> mtk-mmc: adding brackets to prevent warnings

This can be done later. Please drop it from the patchset,

Reply:
Ok